pyproject.toml

```diff
@@ -22,12 +22,12 @@ classifiers = [
     "Topic :: Scientific/Engineering",
     "Topic :: Utilities",
 ]
-dependencies = ["bpack[bs,np]", "numpy", "tqdm"]
+dependencies = ["bpack[bs,np]", "numpy", "tqdm","bitstruct","bpack"]
```

Comment:
please note that the dependency on `bpack` is already there, and `bpack[bs,...]` automatically triggers the installation of `bitstruct`, so no change is needed in this case

Reply:
Resolved